roles/network_plugin/calico/templates/cni-calico.conflist.j2

```diff
@@ -31,16 +31,19 @@
       "ipam": {
         "type": "calico-ipam",
 {%   if enable_dual_stack_networks %}
+{%      if calico_cni_pool_ipv6 %}
         "assign_ipv6": "true",
         "ipv6_pools": ["{{ calico_pool_cidr_ipv6 | default(kube_pods_subnet_ipv6) }}"],
+{%      else %}
+        "assign_ipv6": "true",
+{%      endif %}
 {%   endif %}
+{%      if calico_cni_pool %}
         "assign_ipv4": "true",
         "ipv4_pools": ["{{ calico_pool_cidr | default(kube_pods_subnet) }}"]
-      },
-{% endif %}
-{% if calico_allow_ip_forwarding %}
-      "container_settings": {
-        "allow_ip_forwarding": true
```

Comment:
Why do we need to remove this
```
"allow_ip_forwarding": true
```
?

Reply:
Sorry, I don't know what happened. I didn't want to remove this section from template. How I solve this problem? Do I remove this PR request and create new without this mistake?

Reply:
I added new commit with correct message, if this ok for you.